CHANGELOG.md

```diff
@@ -115,6 +115,33 @@ All notable, unreleased changes to this project will be documented in this file.
 
 - Don't run plugins when calculating checkout's total price for available shipping methods resolution - #9121 by @rafalp
   - Use either net or gross price depending on store configuration.
+- Update webhook payload for `list_shipping_methods_for_checkout`
```

Comment:
Please add a reference number to PR. As it is done for the rest of the records

Reply:
Resolved in recent commit.